src/tests/youtube/Cobalt_URL_001.test.js

```diff
@@ -0,0 +1,39 @@
+import { pluginDeactivate, pluginActivate, getCobaltUrl } from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Cobalt URL - 001',
+  description: 'Default URL on Cobalt',
+  context: {
+    url: 'https://www.youtube.com/tv',
+  },
+  steps: [
+    {
+      description: 'Deactivate Cobalt browser',
+      test: pluginDeactivate,
+      params: constants.youTubePlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Activate Cobalt browser',
+      test: pluginActivate,
+      params: constants.youTubePlugin,
+      assert: 'resumed',
+    },
+    {
+      description: 'Get the default URL Loaded on Cobalt',
+      sleep: 10,
+      test() {
+        return getCobaltUrl.call(this)
```

Comment:
Does this test actually work? I'm not sure if the URL functionality is available on Cobalt?

Reply:
Yeah Wouter. This is working and it shows 'https://www.youtube.com/tv'.